server/server.go

```diff
@@ -505,6 +505,59 @@ func (server HTTPServer) readAggregatorReportForClusterID(
 	return aggregatorResponse.Report, true
 }
 
+// readAggregatorReportMetainfoForClusterID reads report metainfo from Aggregator,
+// handles errors by sending corresponding message to the user.
+// Returns report and bool value set to true if there was no errors
+func (server HTTPServer) readAggregatorReportMetainfoForClusterID(
+	orgID ctypes.OrgID, clusterID ctypes.ClusterName, userID ctypes.UserID, writer http.ResponseWriter,
+) (*ctypes.ReportResponseMetainfo, bool) {
+	aggregatorURL := httputils.MakeURLToEndpoint(
+		server.ServicesConfig.AggregatorBaseEndpoint,
+		"organizations/{org_id}/clusters/{cluster}/users/{user_id}/report/info", //ira_server.ReportMetainfoEndpoint,
```

Comment:
hardcoded URL

Reply:
ah yup, the aggregator were not updated, I'll fix it soon

Reply:
fixed